source/includes/Visit/visits.md

```diff
@@ -155,13 +156,18 @@ GET /visit/:target_visit_uuid/attribute/:target_attribute_uuid
      returned.
 
 ## Create an attribute subresource with properties
-```console
+
+```shell
+
+1. Make sure the `attributeType UUID` is present to avoid error status codes.
+
 POST visit/:target_visit_uuid/attribute 
 {
-    "attributeType": "target_attribute_type_uuid",
-    "value": "value_for_the_attriute"
+    "attributeType": "19a9de73-4d0f-48e4-be7b-b35fe0f8586d",
+    "value": "stable"
 }
```

Comment:
@Akayeshmantha this request is giving internal server error 500, I have checked with the type UUID also it is present on the server

Reply:
@Akayeshmantha I will check this PR with dakiwaya suggestions and again see if it works